client/apps/city_issues/forms/forms.py

```diff
@@ -0,0 +1,37 @@
+"""Forms models"""
+from django.core.validators import MaxValueValidator, MinValueValidator
+from django.forms import (
+    CharField, FloatField, ModelChoiceField, ModelForm,
+    Textarea, TextInput)
+
+from city_issues.models import Issues, Category
+
+
+class EditIssue(ModelForm):
+    """Edit issue form."""
+    name = CharField(
+        min_length=5,
+        max_length=50,
+        widget=TextInput({'size': 50}),
+        required=True)
+
+    description = CharField(
+        min_length=5,
+        max_length=50,
```

Comment:
I think `max_length=50` is not enough for description

Reply:
for the test version is enough